### ISSUE TITLE
processing a glTF file with KHR_materials_ior might result in incorrect IOR value

### ISSUE THREAD

**Issue by robertoranon** [donmccurdy/glTF-Transform#862]
Per [KHR_materials_ior spec](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_ior/README.md), the ior value is not required (it should then be taken as a default of 1.5).

I've found that, when I process a glTF file with KHR_materials_ior and where the ior value is not specified, the output glTF has ior set to 0, instead of 1.5 - probably due to an undefined value (?!) or an incorrect default value.

The expected behavior should be to save the output file again with an unspecified IOR value,  or, alternatively, the default 1.5 value.

Thanks!

**Comment by donmccurdy** [comment]
Thanks @robertoranon! You're correct, I've put in the wrong default value. Would you be interested in doing a PR for the correction? I believe the only required change would be here:

https://github.com/donmccurdy/glTF-Transform/blob/32dc66e8d5e2019f52bdcc6dbb0043d837007708/packages/extensions/src/khr-materials-ior/ior.ts#L25-L27

**Comment by robertoranon** [comment]
yes, of course! I'm not very familiar yet with the codebase, so I did not propose a PR, but will happily do it!

**Comment by robertoranon** [comment]
I created the PR: https://github.com/donmccurdy/glTF-Transform/pull/863

**Comment by donmccurdy** [comment]
Fixed by https://github.com/donmccurdy/glTF-Transform/pull/863, thanks!
